packages/spacecat-shared-data-access/src/models/importer/import-constants.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/**
+ * Supported Import Options.
+ */
+export const ImportOptions = {
+  ENABLE_JAVASCRIPT: 'enableJavascript',
+  PAGE_LOAD_TIMEOUT: 'pageLoadTimeout',
+};
```

Comment:
@blefebvre @swetabar without SAVE_AS_DOCX: 'saveAsDocx' option here, existing imports will fail to load as we load the records from the database. Let's chat about what we want to do here.

Reply:
I also just ran across a option for `hasCustomImportJs` which is failing to load records now.

Reply:
And just found a job with `scrollToBottom`.

Reply:
_Oh also found `hasCustomHeaders`._

Reply:
Yes - now we have both `hasCustomImportJs` and `hasCustomHeaders`.

Reply:
> We can either clean up records in the db that have values which are not valid options at this point

I'd like to +1 the idea to remove all the existing records. I think it makes sense to completely remove everything from the import-jobs and import-urls tables to start fresh.